spec/features/position_deletion_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe 'deleting a position' do
+  let!(:department) { create :department }
+  let!(:base_attributes) do
+    { name: 'A position',
+      department: department,
+      default_interview_location: 'UMTS' }
+  end
+  let!(:position) { create :position, base_attributes }
+  before :each do
+    when_current_user_is :staff, integration: true
+    visit edit_position_path(position)
+  end
+  it 'deletes the position in question' do
+    expect { click_on "Remove #{position.name} (#{position.department.name})" }
```

Comment:
Since the button clicking is common to these tests, perhaps abstract it into a `let` block?